var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -492,15 +494,19 @@ def nvariant_bool(feature):
             cmake_args.append("-DPARAVIEW_ENABLE_PYTHON:BOOL=OFF")
 
         if "+mpi" in spec:
-            cmake_args.extend(
-                [
-                    "-DPARAVIEW_USE_MPI:BOOL=ON",
-                    "-DMPIEXEC:FILEPATH=%s/bin/mpiexec" % spec["mpi"].prefix,
-                    "-DMPI_CXX_COMPILER:PATH=%s" % spec["mpi"].mpicxx,
-                    "-DMPI_C_COMPILER:PATH=%s" % spec["mpi"].mpicc,
-                    "-DMPI_Fortran_COMPILER:PATH=%s" % spec["mpi"].mpifc,
-                ]
-            )
+            mpi_args = [
+                "-DPARAVIEW_USE_MPI:BOOL=ON",
+                "-DMPIEXEC:FILEPATH=%s/bin/mpiexec" % spec["mpi"].prefix,
+            ]
+            if not sys.platform == "win32":
+                mpi_args.extend(
+                    [
+                        "-DMPI_CXX_COMPILER:PATH=%s" % spec["mpi"].mpicxx,
+                        "-DMPI_C_COMPILER:PATH=%s" % spec["mpi"].mpicc,
+                        "-DMPI_Fortran_COMPILER:PATH=%s" % spec["mpi"].mpifc,
+                    ]
+                )
```

Comment:
(no change required) this reminds me of several other common patterns for cmake arguments that could be pulled into the CMake package itself (ROCm support is another example).

Reply:
I'll make a note and next time I'm modifying the CMake I'll see if I can refactor this.